client/my-sites/comments/comment/comment-author.jsx

```diff
@@ -0,0 +1,111 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Emojify from 'components/emojify';
+import ExternalLink from 'components/external-link';
+import Gravatar from 'components/gravatar';
+import { convertDateToUserLocation } from 'components/post-schedule/utils';
+import { gmtOffset, timezone } from 'lib/site/utils';
+import { urlToDomainAndPath } from 'lib/url';
+import { getSiteComment } from 'state/selectors';
+import { getSite } from 'state/sites/selectors';
+import { getAuthorDisplayName, getGravatarUser, getPostTitle } from './utils';
+
+export const CommentAuthor = ( {
+	authorAvatarUrl,
+	authorDisplayName,
+	authorUrl,
+	commentDate,
+	commentType,
+	commentUrl,
+	isExpanded,
+	moment,
+	postTitle,
+	site,
+} ) => {
+	const gravatarUser = getGravatarUser( authorAvatarUrl, authorDisplayName );
```

Comment:
We can probably move this over to connect and expose the gravatarUser and authorDisplayName as props